apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -112,21 +112,11 @@ export class SendMessagePush extends SendMessageBase {
     delete messagePayload.attachments;
 
     if (!pushChannels.length) {
-      await this.createExecutionDetails.execute(
-        CreateExecutionDetailsCommand.create({
-          ...CreateExecutionDetailsCommand.getDetailsFromJob(command.job),
-          detail: DetailEnum.SUBSCRIBER_NO_ACTIVE_CHANNEL,
-          source: ExecutionDetailsSourceEnum.INTERNAL,
-          status: ExecutionDetailsStatusEnum.FAILED,
-          isTest: false,
-          isRetry: false,
-        })
-      );
+      await this.sendErrors(pushChannels, command, notification);
```

Comment:
Forgot to mention we can avoid to pass `pushChannels` to this method as we are doing the check before and it is not needed to do it inside.

Reply:
@p-fernandez done, pushing in a moment